NetBox Attachments Plugin

Addition of https://github.com/Kani999/netbox-attachments

[Pipeline](https://github.com/waja/docker-netbox-plugins/actions/runs/9844978498/job/27179632897?pr=8) has failures. For example:

```
FAIL: test_missing_filters (extras.tests.test_filtersets.TagTestCase.test_missing_filters)
Check for any model fields which do not have the required filter(s) defined.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/netbox/netbox/utilities/testing/filtersets.py", line 138, in test_missing_filters
    self.assertIn(
AssertionError: 'netbox_attachment_id' not found in odict_keys(['id', 'name', 'slug', 'color', 'description', 'object_types', 'created', 'last_updated', 'created_by_request', 'updated_by_request', 'modified_by_request', 'q', 'content_type', 'content_type_id', 'for_object_type_id', 'id__n', 'id__lte', 'id__lt', 'id__gte', 'id__gt', 'id__empty', 'name__n', 'name__ic', 'name__nic', 'name__iew', 'name__niew', 'name__isw', 'name__nisw', 'name__ie', 'name__nie', 'name__empty', 'slug__n', 'slug__ic', 'slug__nic', 'slug__iew', 'slug__niew', 'slug__isw', 'slug__nisw', 'slug__ie', 'slug__nie', 'slug__empty', 'color__n', 'color__ic', 'color__nic', 'color__iew', 'color__niew', 'color__isw', 'color__nisw', 'color__ie', 'color__nie', 'color__empty', 'description__n', 'description__ic', 'description__nic', 'description__iew', 'description__niew', 'description__isw', 'description__nisw', 'description__ie', 'description__nie', 'description__empty', 'object_types__n', 'created__n', 'created__lte', 'created__lt', 'created__gte', 'created__gt', 'created__empty', 'last_updated__n', 'last_updated__lte', 'last_updated__lt', 'last_updated__gte', 'last_updated__gt', 'last_updated__empty']) : No filter defined for netbox_attachment_id (netboxattachment)!
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -20,7 +20,7 @@ RUN apt-get -q update; apt-get -qy upgrade && \
   SECRET_KEY="dummydummydummydummydummydummydummydummydummydummy" /opt/netbox/venv/bin/python /opt/netbox/netbox/manage.py collectstatic --no-input && \
   # Activate plugins
   cat /etc/netbox/config/plugins.py && \
-  echo 'PLUGINS = ["netbox_qrcode","netbox_interface_synchronization","netbox_reorder_rack","netbox_topology_views"]' >> /etc/netbox/config/plugins.py && \
+  echo 'PLUGINS = ["netbox_qrcode","netbox_interface_synchronization","netbox_reorder_rack","netbox_topology_views","netbox_attachments"]' >> /etc/netbox/config/plugins.py && \
   # Cleanup
   apt-get purge \
       --yes -qq \

--- a/plugin_requirements.txt
+++ b/plugin_requirements.txt
@@ -2,3 +2,4 @@ git+https://github.com/netbox-community/netbox-qrcode.git@v0.0.17
 git+https://github.com/netbox-community/netbox-reorder-rack.git@v1.1.3
 git+https://github.com/NetTech2001/netbox-interface-synchronization.git@4.1.6
 git+https://github.com/netbox-community/netbox-topology-views@v4.2.0
+git+https://github.com/Kani999/netbox-attachments.git@7.1.0

